interface/fonts/fonts_datum.dm

```diff
@@ -0,0 +1,48 @@
+/// A font datum, it exists to define a custom font to use in a span style later.
+/datum/font
+	/// Font name, just so people know what to put in their span style.
+	var/name
+	/// The font file we link to.
+	var/font_family
+	/// Font features and metrics
+	/// Generated by Lummox's dmifontsplus (https://www.byond.com/developer/LummoxJR/DmiFontsPlus)
+	var/list/metrics
+	var/height      	// total height of a line
+	var/ascent      	// distance above baseline (including whitespace)
+	var/descent     	// distance below baseline
+	var/avgwidth    	// average character width
+	var/maxwidth    	// maximum character width
+	var/overhang    	// extra width, such as from italics, for a line
+	var/inleading   	// internal leading vertical space, for accent marks
+	var/exleading   	// external leading vertical space, just plain blank
+	var/defchar     	// default character (for undefined chars)
+	var/start       	// first character in metrics
+	var/end				// last character in metrics
```

Comment:
This is not how we comment these.

Make these comments before each variable with `///`.